src/Didomi.ts

```diff
@@ -314,7 +314,7 @@ export const Didomi = {
    *  Get the user consent status.
    *  @returns: status that represents user consent.
    */
-  getUserStatus: (): Promise<any> => RNDidomi.getUserStatus(),
+  getUserStatus: (): Promise<UserStatus> => RNDidomi.getUserStatus(),
```

Comment:
So this is enough to fix the issue?

Reply:
yep.
Just to be sure I added new tests https://github.com/didomi/react-native/pull/56/commits/9d9e5d3fe4391e5f10f91004fc4e9ffc3ac3b83e